src/GitHub.App/Models/Account.cs

```diff
@@ -15,6 +15,8 @@ public class Account : ReactiveObject, IAccount
     {
         BitmapSource avatar;
 
+        IObservable<BitmapSource> BitmapSource;
+        private IDisposable _bitmapSourceSubscription;
```

Comment:
Private fields should be lowercase with no underscore prefix. Feel free to use `this` during constructor assignment if the constructor argument name matches the field it's being assigned to.

We also don't use the `private` keyword in the codebase (it is not by any means a consensual choice, but we're following it for consistency)